packages/react-sdk/README.md

```diff
@@ -22,7 +22,7 @@ If no explicit feature definitions are provided, there will be no types checked
 **Example:**
 
 ```tsx
-import "@bucketco/react-sdk"
+import "@bucketco/react-sdk";
```

Comment:
Do we need to update the docs page as well?

Reply:
great point, done